### DISCUSSION TITLE
How to visit "unsafe" https websites?

### DISCUSSION THREAD

**Discussion by cihe13375** [Eloston/ungoogled-chromium#1972] [Q&A] [answered]
Sometimes I need to visit websites with https incorrectly configured (expired certificates, wrong hostname, etc.). In the original Chrome there's a warning for such sites, but I can click Advanced and proceed to the website anyway. However in ungoogled-chromium they're blocked with the "This site can’t be reached" page, and I can't find an option to add exceptions for them. I'm wondering how can I visit them in ungoogled-chromium.

Example websites:
https://expired.badssl.com/
https://wrong.host.badssl.com/

**Comment by cihe13375** [comment]
Nevermind I fixed it by downgrading to v101 (but the official Google Chrome v102 works idk why)

**Comment by Ahrotahn** [comment, marked as answer]
This is a regression from the last version update due to me removing the security interstitial.  I have a fix I'll submit once I've done a clean build to be sure I haven't caused any other problems.
